components/NavBar.tsx

```diff
@@ -0,0 +1,47 @@
+import { AppBar, Toolbar, Typography, Button, makeStyles } from '@material-ui/core'
+import React, { useContext } from 'react'
+import NextLink from 'next/link'
+
+import ElevationScroll from './ElevationScroll'
+import { AppContext } from './AppContext'
+
+interface Props {
+	title: string
+}
+
+const useStyles = makeStyles({
+	title: {
+		flexGrow: 1,
+	},
+})
+
+export default function NavBar(props: Props): JSX.Element {
+	const { themeType, setThemeType } = useContext(AppContext)
+	const classes = useStyles()
+
+	const handleThemeInvert = (): void => {
+		if (setThemeType) {
+			setThemeType(themeType === 'dark' ? 'light' : 'dark')
+		}
+	}
+
+	return (
+		<ElevationScroll>
+			<AppBar position="sticky">
+				<Toolbar>
+					<NextLink href="/">
+						<Typography variant="h5" className={classes.title}>
+							{props.title}
+						</Typography>
+					</NextLink>
+					<NextLink href="/about">
+						<Button color="inherit">About</Button>
```

Comment:
`NextLink`->`Button` should have `component="a"` to avoid

```
Warning: You're using a string directly inside <Link>. This usage has been deprecated. Please add an <a> tag as child of <Link>
```

...thinking about it, I wonder if this also affects our `<MuiLink component={NextLink}` memes? It doesn't seem to luckily.

Reply:
`<NextLink` might also need `passHref` idk